src/VectorFiber.jl

```diff
@@ -0,0 +1,279 @@
+
+"""
+    VectorSpaceFiberType{TVS<:VectorSpaceType}
+
+`FiberType` of a [`FiberBundle`](@ref) corresponding to [`VectorSpaceType`](@ref)
+`TVS`.
+"""
+struct VectorSpaceFiberType{TVS<:VectorSpaceType} <: FiberType
+    fiber::TVS
+end
+
+function Base.show(io::IO, vsf::VectorSpaceFiberType)
+    return print(io, "VectorSpaceFiberType($(vsf.fiber))")
+end
+
+const TangentFiberType = VectorSpaceFiberType{TangentSpaceType}
+
+const TangentFiber = VectorSpaceFiberType{TangentSpaceType}(TangentSpace)
+
+"""
+    VectorBundleFibers{TVS,TM}
+
+Alias for [`BundleFibers`](@ref) when the fiber is a vector space.
```

Comment:
does BundleFibers still exist? (You might notice I am a bit lost in the zoo of types involving the 3 words bundle-fiber-vector and arbitrary permutations of these and sometimes added with a `Type` at the end).